RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitOOTECAL_cff.py

```diff
@@ -1,6 +1,34 @@
 import FWCore.ParameterSet.Config as cms
 from RecoParticleFlow.PFClusterProducer.particleFlowRecHitECAL_cfi import *
 
-particleFlowOOTRecHitECAL = particleFlowRecHitECAL.clone()
-particleFlowOOTRecHitECAL.producers[0].qualityTests[1].timingCleaning = False
-particleFlowOOTRecHitECAL.producers[1].qualityTests[1].timingCleaning = False
+particleFlowRecHitOOTECAL = particleFlowRecHitECAL.clone()
+particleFlowRecHitOOTECAL.producers[0].qualityTests[1].timingCleaning = False
+particleFlowRecHitOOTECAL.producers[1].qualityTests[1].timingCleaning = False
+
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+## EB
+run2_miniAOD_80XLegacy.toModify(
+    particleFlowRecHitOOTECAL.producers[0], 
+    src = "reducedEcalRecHitsEB",
+    srFlags = ""
+)
+run2_miniAOD_80XLegacy.toModify(
+    particleFlowRecHitOOTECAL.producers[0].qualityTests[0], 
+    name = "PFRecHitQTestThreshold",
+    threshold = cms.double(0.08),
```

Comment:
Ah, I didn't make this change with these commits. Where can I see the era-dependent (i.e. for Run2_2016) thresholds?  It was not present in: RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitECAL_cfi.py